config/systems/eiger.py

```diff
@@ -116,9 +116,7 @@
         {
             'name': 'PrgEnv-aocc',
             'target_systems': ['eiger'],
-            'modules': ['cray', 'PrgEnv-aocc'],
-            'features': ['serial', 'openmp', 'mpi', 'cuda',
-                         'hdf5', 'netcdf-hdf5parallel', 'pnetcdf', 'openmp']
+            'modules': ['cray', 'PrgEnv-aocc']
```

Comment:
Do not change this file

Reply:
ups... this must have been caused by the rebase into main